### ISSUE TITLE
Improve picture of the projection onto a line

### ISSUE THREAD

**Issue by nbro** [pennal/LinAlgNotes#14]
Chapter 5, page 41, there's a picture of a projection of a vector onto a line, but actually that picture is not correct.

This is the current image:

![screen shot 2015-06-08 at 18 59 04](https://cloud.githubusercontent.com/assets/9349000/8040394/7891d8ae-0e10-11e5-9424-f212d5be8ee5.png)

But it should look something like this:

![screen shot 2015-06-08 at 18 59 04](https://cloud.githubusercontent.com/assets/9349000/8040460/fd256d1a-0e10-11e5-9dfb-0cb57716ef83.png)

Basically is missing the vector `p`, the projection of `b` onto the line spanned by `a`. Note that the line spanned by `a` should also be added... Of course the vector `a` remains there anyway...Also `b` is too long, and should arrive just up to the _error_ vector `e` , the vector that is perpendicular to the line spanned by `a`, so also the second picture is not correct, sorry...

**Comment by pennal** [comment]
I'll take care of it one of these days, not really a top priority right now
